### ISSUE TITLE
how to change the default behaviour,disabled="true" to false for a grommet submit button

### ISSUE THREAD

**Issue by cristyalex** [grommet/grommet#1758]
### Actual Behavior
Even if I give disabled="false",
<Button label="Submit" type="submit" primary={true} onClick={handleSubmit} disabled="false"/>
![image](https://user-images.githubusercontent.com/34138120/33471770-dc6070f8-d694-11e7-8a60-96472e27e9fc.png)

by default it takes disabled="true"
![image](https://user-images.githubusercontent.com/34138120/33471719-abdd755c-d694-11e7-99c9-b1a196836b49.png)

### Steps to Reproduce
 1. 
 2. 
 3. 

### Your Environment

* Grommet version:
* Browser Name and version:
* Operating System and version (desktop or mobile):

**Comment by quaqua** [comment]
Have a look in  the [Button component](https://github.com/grommet/grommet/blob/68fb633e9fcd2d7e571f2760e3414775cd4497ff/src/js/components/Button.js#L177) source code. The condition says:

`disabled={!onClick && !adjustedHref && ['reset', 'submit'].indexOf(type) === -1}`

It ignores your `disabled="false"` or even the better approach `disabled={false}`.

Also, in #1681 a comment says, it is a design issue and be fixed in 2.0

So basically, the reason for your disabled submit button might be, that your `handleSubmit` function is undefined. That was the case on my side when I was experiencing that behaviour. That's why I'm commenting on this.
